charts/qiskit-serverless/charts/gateway/templates/deployment.yaml

```diff
@@ -74,6 +74,8 @@ spec:
             httpGet:
               path: /liveness
               port: http
+            initialDelaySeconds: 60
+            periodSeconds: 20
```

Comment:
Even if we don't go with this PR, we should add these configs to ensure we aren't failing the gateway prematurely...

Reply:
I see, maybe this is the reason cause the gateway restart some times when we deploy it, could be?